TemplateApplication/Supporting Files/TemplateApplication.docc/TemplateApplication.md

```diff
@@ -18,73 +18,73 @@ The Spezi Template Application demonstrates using the [Spezi](https://github.com
+            [HealthKit Access](https://spezi.health/SpeziTemplateApplication/documentation/templateapplication/healthkitpermissions)
         }
     }
     @Column(size: 1) {
         @Image(source: "HealthKitSheet", alt: "Permissions screen of the HealthKit framework") {
-            Granular HealthKit Share Control.
+            [Granular HealthKit Share Control](https://spezi.health/SpeziTemplateApplication/documentation/templateapplication/healthkitpermissions)
         }
     }
     @Column(size: 1) {
-        @Image(source: "Notifications", alt: "Onboarding screen showing the Notifications permission screen.") {
-            Trigger Local Notifications.
+        @Image(source: "Schedule", alt: "A screen displaying the Scheduler UI.") {
+            [Schedule Tasks](https://spezi.health/SpeziTemplateApplication/documentation/templateapplication/scheduleview)
         }
     }
 }
 @Row(numberOfColumns: 3) {
     @Column(size: 1) {
-        @Image(source: "Schedule", alt: "A screen displaying the Scheduler UI.") {
-            Schedule Tasks.
+        @Image(source: "Notifications", alt: "Onboarding screen showing the Notifications permission screen.") {
+            [Trigger Local Notifications](https://spezi.health/SpeziTemplateApplication/documentation/templateapplication/notificationpermissions)
         }
     }
     @Column(size: 1) {
         @Image(source: "Questionnaire", alt: "A screen showing a questionnaire using ResearchKit.") {
-            Display Questionnaires.
+            [Display Questionnaires](https://spezi.health/SpeziTemplateApplication/documentation/templateapplication/scheduleview)
         }
     }
     @Column(size: 1) {
         @Image(source: "ScheduleComplete", alt: "The scheduler screen showing the completed UI") {
-            Keep Track of Tasks.
+            [Keep Track of Tasks](https://spezi.health/SpeziTemplateApplication/documentation/templateapplication/scheduleview)
         }
     }
 }
 @Row(numberOfColumns: 3) {
     @Column(size: 1) {
         @Image(source: "Contacts", alt: "A screen displaying the Contact UI.") {
-            Contact Information.
+            [Contact Information](https://spezi.health/SpeziTemplateApplication/documentation/templateapplication/contacts)
         }
     }
     @Column(size: 1) {
-        @Image(source: "Account", alt: "A screen displaying the current user account information.") {
-            Account Overview.
+        @Image(source: "License", alt: "License information to list all used Swift Packages") {
+            [License Information](https://spezi.health/SpeziTemplateApplication/documentation/templateapplication/licensetype)
         }
     }
     @Column(size: 1) {
-        @Image(source: "License", alt: "License information to list all used Swift Packages") {
-            License Information.
+        @Image(source: "Account", alt: "A screen displaying the current user account information.") {
+            Account Overview.
         }
```

Comment:
I resolved a merge conflict here; we might want to double-check and adjust this.